tests/nnstreamer_query/runTest.sh

```diff
@@ -45,7 +47,7 @@ function waitformarker {
   for i in $(seq 1 ${TIMEOUT_SEC})
   do
     if [ -f 'marker.log' ]; then
-      markersize=$(stat -c%s marker.log)
+      markersize=$([[ ${SO_EXT} == "dylib" ]] && stat -f%z marker.log || stat -c%s marker.log)
```

Comment:
There's similar script at https://github.com/nnstreamer/nnstreamer/blob/main/tests/nnstreamer_grpc/runTest.sh

Reply:
@myungjoo We've already defined ```StatCmd_GetSize``` in ```ssat-api.sh```. Thus, I revised this PR as well as #3825 to use ```StatCmd_GetSize``` .